### PR TITLE
Rspec saturday branch

### PR DESCRIPTION
**Pull request by LawrenceWhalen**
## Changes
  Add test coverage
  Add tests for all queries

## Checklist
- [x] Tests written where required
- [x] All tests passing
- [ ] SimpleCov at 100%

closes #

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -52,6 +52,7 @@ GEM
     concurrent-ruby (1.1.9)
     crass (1.0.6)
     diff-lcs (1.4.4)
+    docile (1.4.0)
```
Comment (Aferencz1987) [inline Gemfile.lock:55]: what is docile?

Comment (LawrenceWhalen) [reply Gemfile.lock:55]: added as part of the simplecov gem

Comment (ahenzler) [reply Gemfile.lock:55]: same as aboce

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 require 'pry'
```
Comment (ahenzler) [inline spec/rails_helper.rb:6]: glad we added simplecov

**Review by Aferencz1987 — APPROVED** [review]

**Review by ahenzler — APPROVED** [review]
the testing is really well done.

**Pull request merged**